src/current/v24.1/begin-transaction.md

```diff
@@ -7,15 +7,14 @@ docs_area: reference.sql
 
 The `BEGIN` [statement]({% link {{ page.version.version }}/sql-statements.md %}) initiates a [transaction]({% link {{ page.version.version }}/transactions.md %}), which either successfully executes all of the statements it contains or none at all.
 
-{{site.data.alerts.callout_danger}}
-When using transactions, your application should include logic to [retry transactions]({% link {{ page.version.version }}/transactions.md %}#transaction-retries) that are aborted to break a dependency cycle between concurrent transactions.
+{{site.data.alerts.callout_info}}
+When running under the default [`SERIALIZABLE`]({% link {{ page.version.version }}/demo-serializable.md %}) isolation level, your application should [use a retry loop to handle transaction retry errors]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#transaction-retry-errors) that can occur under [contention]({{ link_prefix }}performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention).
```

Comment:
For all versions, maybe use `#transaction-contention` heading anchor because `understanding-and-avoiding-transaction-contention` anchor cuts off initial line of section:
<img width="1052" alt="Screenshot 2024-12-12 at 12 12 33 PM" src="https://github.com/user-attachments/assets/d40c7a21-e586-405c-b562-44d2e3846dbf" />